### PR TITLE
chore: Add container_analysis library to main readme

### PR DESCRIPTION
**Pull request by dazuma**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -320,6 +321,31 @@ end
...
+- [google-cloud-container_analysis on RubyGems](https://rubygems.org/gems/google-cloud-container_analysis)
+- [Container Analysis documentation](https://cloud.google.com/container-registry/docs/container-analysis/)
+
+#### Quick Start
+
+```sh
+$ gem install google-cloud-container_analysis
+```
+
+#### Preview
```
Comment (quartzmo) [inline README.md]: This example is not present in [google-cloud-container_analysis/README.md)](https://github.com/googleapis/google-cloud-ruby/blob/master/google-cloud-container_analysis/README.md). Do you want to add it there as well, or omit it here?

File: README.md [hunk 2]
```diff
@@ -320,6 +321,31 @@ end
...
+
+```ruby
+require "google/cloud/container_analysis"
+
+grafeas_client = Grafeas.new
+parent = Grafeas::V1::GrafeasClient.project_path my_project_id
```
Comment (quartzmo) [inline README.md]: As a rule, code examples should be executable. In the handwritten packages we try to always use examples that are copied from API docs that are under doctest. Maybe  s/my_project_id/"my-project"/ ?

File: README.md (hunk 1 above)
Comment (dazuma) [reply README.md]: Added it to the container_analysis readme. (This required me to remove the readme from synth, but meh.)

File: README.md (hunk 2 above)
Comment (dazuma) [reply README.md]: Done.

**Pull request merged**